packages/dustkit/src/rpc/schemas.ts

```diff
@@ -32,6 +32,19 @@ export type ClientRpcSchema = RpcSchema.From<
           }
       >;
     }
+  | {
+      Request: {
+        method: "dustClient_playerPosition";
```

Comment:
nit: should be `getPlayerPosition`, ie `get` prefix to clearly indicate what its doing